docs/concepts/llm_convenience_wrappers.md

```diff
@@ -18,14 +18,14 @@ You can initialize an [`OpenAIChat`](../api/chat/models.md#mirascope.chat.models
 ```python
 from mirascope import OpenAIChat, Prompt
 
-class RecipePrompt:
-		"""
-		Recommend recipes that use {ingredient} as an ingredient
-		"""
+class RecipePrompt(Prompt):
+	"""
+	Recommend recipes that use {ingredient} as an ingredient
+	"""
 
-		ingredient: str
+	ingredient: str
 
-chat = OpenAIChat()
+chat = OpenAIChat(api_key="YOUR_OPENAI_API_KEY")
 res = chat.create(RecipePrompt(ingredient="apples"))
```

Comment:
do you think we should separate these out? e.g.

```python
chat = OpenAIChat()
prompt = RecipePrompt(ingredient="apples")
res = chat.create(prompt)
```

Especially since later on we'll likely want to do stuff with `prompt` anyway, so good to have it cleanly separated as a separate variable.

Reply:
i dont see anything wrong with this example. i would write it as is, and then update to use variable if i need to reuse.